scripts/run_e2e_tests.sh

```diff
@@ -107,6 +107,33 @@ if [ -d "../protractor-screenshots" ]; then
   rm -r ../protractor-screenshots
 fi
 
+# Pass different suites for tests.
+# For performing a full test, no argument is required.
+# For performing tests on editors, use --suite=editor
+# For performing tests on user management, use --suite=usermanagement
+# For performing miscellaneous tests, use --suite=misc
+SUITE="full"
```

Comment:
Can you instead combine this with lines 139-165 (i.e. the 'SHARDING=true ... done' block)? That whole block is meant to parse the args to the script. It seems unnecessary to duplicate the code so much here.